packages/fluttium_runner/lib/src/fluttium_runner.dart

```diff
@@ -93,24 +93,24 @@ class FluttiumRunner {
   }
 
   void _convertFlowToVars() {
-    String sanitize(String text) => text.replaceAll("'", r"\'");
+    String sanitize(String text) => text.replaceAll("'''", r"\'''");
 
     _vars.addAll({
-      'flowDescription': sanitize(flow!.description),
+      'flowDescription': '''sanitize(flow!.description)''',
```

Comment:
```suggestion
      'flowDescription': '''${sanitize(flow!.description)}''',
```

I expect this was the wanted result 😛

Reply:
Yeah right... I saw this too late, and expect fixing it before you review 😅